docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -0,0 +1,85 @@
+[role="xpack"]
+[[ml-configuring-alerts]]
+= Configuring {anomaly-detect} alerts
+
+{anomaly-detect-cap} alerts run scheduled checks on an {anomaly-job} or a group 
+of jobs to detect anomalies with certain conditions. If an {anomaly} meets the 
+conditions, the alert triggers the defined action. For example, you can create 
+an alert that checks an {anomaly-job} in every fifteen minutes for critical 
+anomalies and notify you in an email. This page helps you to configure an 
+{anomaly-detect} alert. To learn more about alerts in the {stack}, refer to 
+{kibana-ref}/alerting-getting-started.html#alerting-getting-started[Alerting and Actions].
+
+
+[[creating-anomaly-alerts]]
+== Creating an alert
+
+{anomaly-detect-cap} alerts can be created in the {anomaly-job} wizard, from the 
+job list or under **Stack management > Alerts and Actions**. On the *Create 
```

Comment:
There are attributes for these labels, since they've been renamed at least once:
```suggestion
job list or under **{stack-manage-app} > {alerts-ui}**. On the *Create 
```